src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -392,3 +405,16 @@ export const getFormSchema = (
     },
   },
 });
+
+const VersionControlEnabledDescription = (
+  <>
+    Enabling Version Control allows{" "}
+    <Link
+      hideExternalIcon
+      href="https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#version-control"
```

Comment:
nit. Place this url in the externalResources file. 
https://github.com/evergreen-ci/spruce/blob/40db57059ee5ca9a1a92237e31002abcbe18b045/src/constants/externalResources.ts